memoryGame.js

```diff
@@ -0,0 +1,158 @@
+    that will allow you to pick two slots out of nine nine slot table and it will check whether 
+    they match and although it wont let you see what it is real value it will 
+    tell you which one is higher and which one is lower of the two values
+    ...The catch is you only have a limited amount of tries...(10)`)
+    
+    let str = rls.question("Please enter four single-digit numbers in a row \n") //will add non repeating later on
+   
+    
+    while( tries > 0){
+        if( correctOriginal(str)){
+            
+            gameArr = gameArrayAssign(arr)
+            console.log("Now your numbers have been duplicated and randomly assigned")
+            console.log("to an empty line with 8 available slots")
+            console.log("Now we are dealing with 8 numbers which are in a random order \n")
+            
+
+            let answer = correctInput(userInput())
+            console.log(answer)
+
+        
+        }else{
+            console.clear()
+            console.log("Lets read the instrctions again")
+            gameLoop()
+        }
+        tries--
+        console.log("OH Oh! You have the following amount of tries: " + tries )
+    }
+
+    restart()    
+}
+
+const startGame = () =>{
+    if(rls.keyInYN("Would you like to play a memory test?")){
+        console.log("Cheers! , Lets kick it")
+       gameLoop()
+     }else {
+       console.log("Not up for it? No worries")
+       endGame()
+     }
+}
+
+const endGame = () => {
+    console.log(" Try harder next time, lets fight aging by playing more memory games!")
+    process.exit()
+}
+
+const correctOriginal =(str) =>{
+    if(str.length === 4){
+    let arrStr = str.split("")
+    arr = arrStr.map(String)
+    console.log(typeof arr)    // to test 
+    console.log(arr)            // to test 
+        for( i = 0; i< arr.length; i++){
+            if(poss.includes(arr[i])){
+
+                for(i = 1 ; i < arr.length; i++){
+
+                    if (arr[0]===arr[i]){
+
+                        console.clear()
+                        console.log("Typo? if not, lets read the fineprint too!")
+                        gameLoop()
+                    } 
+                }
+            }else{
+
+                console.log("Only numbers please now you have to start all over!")
+                gameLoop()
+            }
+        }
+        return arr
+    }    
+    
+}
+
+const gameArrayAssign = (arr) =>{
+    console.log(shuffleArray(arr.concat(arr)))       // to test
+    return shuffleArray(arr.concat(arr))
+     
+}
+
+const shuffleArray = (arr) =>{
+    for ( i = arr.length - 1; i > 0; i--) {
+        let j = Math.floor(Math.random() * (i + 1));
+        let temp = arr[i];
+        arr[i] = arr[j]
+        arr[j] = temp
+        return arr 
+    }
+}
+
+
+const userInput = () => {
+    userAns = rls.question("enter the possible order of the new list of numbers \n")
+    console.log(typeof userAns)             // to test purposes
```

Comment:
I like that you're console logging to check, but try not to push with these still in the code.